src/components/Feed/Events/CollectorsNoteAddedToTokenFeedEvent.tsx

```diff
@@ -114,7 +115,9 @@ export default function CollectorsNoteAddedToTokenFeedEvent({ eventRef, queryRef
           </StyledMediaWrapper>
           <Spacer width={MIDDLE_GAP} />
           <StyledNoteWrapper>
-            <StyledNote>{unescape(event.newCollectorsNote ?? '')}</StyledNote>
+            <StyledNote>
+              <Markdown text={unescape(event.newCollectorsNote ?? '')} inheritLinkStyling={true} />
```

Comment:
nit: if a prop is set to `true`, you can actually just use the name of the prop and nothing else!
```js
<Markdown
  text={unescape(event.newCollectorsNote ?? '')}
  inheritLinkStyling
/>
```